build/bin/vars

```diff
@@ -2,6 +2,9 @@ bin_dir=$(cd $(dirname $0); pwd -P)
 root_dir=$(cd $(dirname $0)/../..; pwd -P)
 tmp_dir=${root_dir}/tmp
 
+# Use s3 cache for PMM server build (build-server-rpm)
+cache=${CACHE:-false}
```

Comment:
The most important thing here is that it works by default for non-Percona people. They won't have the patience to screw around with various config options on their very first attempt to build it, it should just work.

Pretty sure I can come up with an alternative check that doesn't disable cache for existing users though. I've since learned that the aws command we use does not seem to require authentication, so we can instead check for an installed awscli and skip cache download if the `aws` command doesn't exist.

Reply:
Pushed a new commit as per the above.